internal/server/config.go

```diff
@@ -844,7 +844,7 @@ func (s Server) loadAccessKey(db *gorm.DB, identity *models.Identity, provider *
 		}
 
 		accessKey := &models.AccessKey{
-			Name:       fmt.Sprintf("%s-%s", identity.Name, time.Now().Format("20060102150405")),
+			Name:       fmt.Sprintf("%s-%s", identity.Name, time.Now().UTC().Format("2006-01-02T15:04:05.000000")),
```

Comment:
Making this change so the name generated by API and config are consistent. This also fixes a race where the same name can be generated if an operator decides to import two access keys for the same user